sdk/core/Azure.Core/src/Pipeline/Internal/RedirectPolicy.cs

```diff
@@ -8,17 +8,31 @@
 
 namespace Azure.Core.Pipeline
 {
-    internal sealed class RedirectPolicy : HttpPipelinePolicy
+    /// <summary>
+    /// A pipeline policy that detects a redirect response code and resends the request to the
+    /// location specified by the response.
+    /// </summary>
+    public sealed class RedirectPolicy : HttpPipelinePolicy
```

Comment:
If we decide this is the approach we want to take in the Core API, I will move this file out of the `Internal` folder.  I wanted to leave it here for a clean diff in the initial PR.

Reply:
This is completed.